src/widgets/passwordReset/passwordResetWidget.jsx

```diff
@@ -28,16 +28,17 @@ class MainView extends React.Component {
     };
 
     render() {
-        const { showLabels, onError, i18n } = this.props;
+        const { i18n } = this.props;
 
         return <div>
             <Heading>{i18n('passwordReset.title')}</Heading>
             <Info>{i18n('passwordReset.intro')}</Info>
             <PasswordEditorForm
                 handler={this.handleSubmit}
-                showLabels={showLabels}
+                canShowPassword={this.props.canShowPassword}
+                showLabels={this.props.showLabels}
                 onSuccess={this.onSuccess}
-                onError={onError} />
+                onError={this.props.onError} />
```

Comment:
Use the destructuring props here (not `this.props.`)

Reply:
I don't agree, this method comes from the `props`.

Reply:
But why don't you use then
```const { canShowPassword, showLabels, onError, i18n } = this.props;```

Reply:
Why should I?  `onError` is only used once, no need to create a variable.

Reply:
True but in a PR, it's a clean way to directly see when you add or remove a prop within the component.

Reply:
I think it's a subjective perspective and since we haven't fixed yet the code style for the UI SDK, let's discuss it in another PR.

Reply:
Alright nothing to do with this PR, I agree 👍